src/environment.rs

```diff
@@ -0,0 +1,103 @@
+/// Structure to deal with environment variables
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Environment {
+    /// Customized environment variables
+    pub vars: Vec<(String, String)>,
+    /// Define if the structure must inherit
+    pub inherit: bool,
```

Comment:
The members should probably not be `pub`

See https://rust-lang-nursery.github.io/api-guidelines/future-proofing.html#structs-have-private-fields-c-struct-private